src/internal/connector/discovery/api/users.go

```diff
@@ -220,3 +326,31 @@ func validateUser(item any) (models.Userable, error) {
 
 	return m, nil
 }
+
+func convertInterfaceToString(ctx context.Context, data interface{}, userInfo *UserInfo) string {
+	var (
+		ok          bool
+		dataPointer *string
+		value       string
+	)
```

Comment:
These pre-declarations are unnecesssary: the values aren't getting re-used, overwritten, referenced multiple times, nor are we conditionally using their zero value.  It's cleaner in this case to use normal declaration.  eg: datapointer, ok := ...